client/src/scripts/script.js

```diff
@@ -75,62 +71,129 @@ window.addEventListener('DOMContentLoaded', function () {
-          parameter: 'w'
-        },
-        function () { 
-          console.log('w pressed'); 
-          playerMesh.applyImpulse(new BABYLON.Vector3(0, 0, -10), playerMesh.getAbsolutePosition());
-        }
-      )
-    );
-    scene.actionManager.registerAction(
-      new BABYLON.ExecuteCodeAction(
-        {
-          trigger: BABYLON.ActionManager.OnKeyDownTrigger,
-          parameter: 'd'
-        },
-        function () { 
-          console.log('d pressed'); 
-          playerMesh.applyImpulse(new BABYLON.Vector3(-10, 0, 0), playerMesh.getAbsolutePosition());
+
+    //CHANGING DEFAULT KEYBOARD INPUTS.
+
+    function KeyboardController(keys, repeat) {
+
+      
+      var timers = {};
+
+      // When key is pressed and we don't already think it's pressed, call the
+      // key action callback and set a timer to generate another one after a delay
+      //
+      document.onkeydown = function (event) {
+        // console.log(event);
+        // console.log(keys);
+        var key = (event || window.event).keyCode;
+        if (!(key in keys))
+          return true;
+        if (!(key in timers)) {
+          timers[key] = null;
+          keys[key]();
+          if (repeat !== 0)
+            timers[key] = setInterval(keys[key], repeat);
         }
-      )
-    );
-    scene.actionManager.registerAction(
-      new BABYLON.ExecuteCodeAction(
-        {
-          trigger: BABYLON.ActionManager.OnKeyDownTrigger,
-          parameter: 's'
-        },
-        function () { 
-          console.log('s pressed'); 
-          playerMesh.applyImpulse(new BABYLON.Vector3(0, 0, 10), playerMesh.getAbsolutePosition());
+        return false;
+      };
+
+      // Cancel timeout and mark key as released on keyup
+      //
+      document.onkeyup = function (event) {
+        var key = (event || window.event).keyCode;
+        if (key in timers) {
+          if (timers[key] !== null)
+            clearInterval(timers[key]);
+          delete timers[key];
         }
-      )
-    );
+      };
+
+      // When window is unfocused we may not get key events. To prevent this
+      // causing a key to 'get stuck down', cancel all held keys
+      //
+      window.onblur = function () {
+        for (key in timers)
+          if (timers[key] !== null)
+            clearInterval(timers[key]);
+        timers = {};
+      };
+    };
+    then:
+
+    KeyboardController({
+
+      // a key
+      65: () => playerMesh.applyImpulse(new BABYLON.Vector3(5, 0, 0), playerMesh.getAbsolutePosition()),
+
+      // w key
+      87: () => playerMesh.applyImpulse(new BABYLON.Vector3(0, 0, -5), playerMesh.getAbsolutePosition()),
+
+      // d key
+      68: () => playerMesh.applyImpulse(new BABYLON.Vector3(-5, 0, 0), playerMesh.getAbsolutePosition()),
+
+      // s key
+      83: () => playerMesh.applyImpulse(new BABYLON.Vector3(0, 0, 5), playerMesh.getAbsolutePosition())
+
+      // this is the delay between repeats
+    }, 50);
+
+
+    //CONTROLS
```

Comment:
do you mind nuking this old code, we can still pull it from the git history if we need it.